index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
- [ ] As mentioned by the previous reviewer, your Pull Request title and summary/description is still not written professionally, Kindly follow the instructions on [the previous review](https://github.com/rogeliomonterde/hello/pull/1#pullrequestreview-1243113454).
- [ ] There are conflicts in your code files. [Please read through this article to learn to resolve code conflicts](https://akshayranganath.github.io/Git-Pull-Handling-Merge-Conflict/).
- [ ] Kindly go through all the changes requested by other reviewers and make changes accordingly. Thank you